package.json

```diff
@@ -3,8 +3,11 @@
   "version": "1.0.0",
   "description": "this is a workshop about cypress",
   "main": "index.js",
+ 
   "scripts": {
-    "test:open": "cypress open"
```

Comment:
you should not remove this test:open script, since it can help you to test locally, also, to fix the CI, you should add the lint command here. See step 4, of section 6: https://github.com/testing-community/cypress-training-doc#6-an%C3%A1lisis-de-c%C3%B3digo-estatico